modules/supported-installation-methods.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * configuration/internal-certificate-authority-rotation-for-rhacs.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="supported-installation-methods_{context}"]
+= Supported installation methods
+
+The following table shows the level of certificate authority (CA) rotation support based on the installation method you use for Central and secured clusters:
+
+.Supported installation methods
+[cols="2,2,2,2",options="header"]
+|===
+|Component |Installation method |CA rotation support |Notes
+
+|Central
+|Operator-installed
+|Full support
+|Migration is fully automated.
+
+|Central
+|Helm or manifest
+|Not supported
+|Use the Operator for automatic CA rotation.
+
+|Secured cluster
+|Operator-installed
+|Full support
+|Migration is fully automated provided the connected Central instance is also Operator-installed.
+
+|Secured cluster
+|Helm or manifest
+|Partial support
+|Can connect to Central with a rotated CA, but can not rotate its own service certificates to the new CA. Manual re-registration is required before the old CA expires.
```

Comment:
```suggestion
|The secured cluster can connect to Central with a rotated CA, but cannot rotate its own service certificates to the new CA. Manual re-registration is required before the old CA expires.
```

Reply:
Check with @vladbologa to make sure I've reworded that accurately but I think that the implied subject is the secured cluster.

Reply:
Yes, the secured cluster is the subject. Central is the one with the rotated CA. I'm not sure if that's clear enough, maybe "connect to a Central that has rotated its CA" would be a bit more clear?